sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventFacadeEjb.java

```diff
@@ -340,6 +342,19 @@ public List<String> deleteEvents(List<String> eventUuids, DeletionDetails deleti
 		return deletedEventUuids;
 	}
 
+	@RightsAllowed({
+		UserRight._CASE_DELETE,
+		UserRight._SYSTEM })
+	public void deleteEventInExternalSurveillanceTool(Event event) throws ExternalSurveillanceToolException {
+
+		if (externalSurveillanceToolGatewayFacade.isFeatureEnabled() && event.getExternalId() != null && !event.getExternalId().isEmpty()) {
```

Comment:
Use StringUtils.isNotBlank instead of combining a null check with an empty check